src/ontology/cl-edit.owl

```diff
@@ -29104,14 +29106,14 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023006 "nuclear bag2 intraf
 AnnotationAssertion(rdfs:label obo:CL_4023006 "static nuclear bag fiber")
 SubClassOf(obo:CL_4023006 obo:CL_4023004)
 
-# Class: obo:CL_4023007 (L2/3 bipolar vip GABAergic interneuron (Mmus))
+# Class: obo:CL_4023007 (L2/3 bipolar VIP GABAergic interneuron (Mmus))
 
-AnnotationAssertion(obo:IAO_0000028 obo:CL_4023007 "L2/3 bipolar vip interneuron (Mus)")
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26420784") obo:IAO_0000115 obo:CL_4023007 "A vip GABAergic cortical interneuron with bipolar morphology, with a soma found in L2/3. L2/3 bipolar vip cells have extending axons across all layers (with preferences for layers II/III and Va) and a dendritic tree that is vertically more restricted than deeper layer vip cells and extend fewer dendrites into the layers outside their home layer (location of soma). L2/3 bipolar vip cells have great variability in firing patterns, though most are continuous adapting. L2/3 bipolar vip cells are more depolarized in their resting state, had less fast rectification, and had smaller after hyperpolarization than deeper vip cells.")
+AnnotationAssertion(obo:IAO_0000028 obo:CL_4023007 "L2/3 bipolar VIP interneuron (Mus)")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26420784") obo:IAO_0000115 obo:CL_4023007 "A VIP GABAergic cortical interneuron with bipolar morphology, with a soma found in L2/3. L2/3 bipolar VIP cells have extending axons across all layers (with preferences for layers II/III and Va) and a dendritic tree that is vertically more restricted than deeper layer vip cells and extend fewer dendrites into the layers outside their home layer (location of soma). L2/3 bipolar VIP cells have great variability in firing patterns, though most are continuous adapting. L2/3 bipolar VIP cells are more depolarized in their resting state, had less fast rectification, and had smaller after hyperpolarization than deeper VIP cells.")
```

Comment:
There is still one vip lower case (they hide well!)
"...restricted than deeper layer vip cells and..."

Reply:
Yes they do! thank you!